doc/api/deprecations.md

```diff
@@ -523,6 +523,12 @@ Type: Runtime
 of V8 5.8. It is replaced by Inspector which is activated with `--inspect`
 instead.
 
+<a id="DEP00XX"></a>
```

Comment:
Ok, I just had noticed that one of the other deprecation PRs already had a number assigned, so I was confused...